src/main.py

```diff
@@ -237,6 +241,56 @@ async def info(interaction: Interaction) -> None:
     await bc.send_stats(interaction)
 
 
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(description="The file to submit."),
+    submit_message: Optional[str] = SlashOption(
+        required=False, description="The submission message text."
+    ),
+) -> None:
+    """Show a preview of a submission's "Now Playing" embed."""
+    await interaction.response.defer(ephemeral=True)
+    user = interaction.user
```

Comment:
I'm not sure if this is really that helpful, especially when `user` is only used once below.

Reply:
Have you tested this since adding the defer to the beginning? It may have timed out if the attachment save took longer than 3 seconds without a defer.

It doesn't make sense looking at this that it's necessary. In general it's a good idea to find the reason for unexpected behavior like this instead of doing something less optional that seems to work